Dockerfile

```diff
@@ -11,6 +11,11 @@ RUN if [ $(command -v apt-get) ]; then \
       yum install -y sudo && \
       usermod -aG wheel logstash; \
     fi
+RUN if [ $(command -v apt-get) ]; then \
+      apt-get install shared-mime-info \
```

Comment:
does it need `apt-get install -y` ?

Reply:
Yep, maybe it's better, thanks for noting!